Makefile.dune

```diff
@@ -37,6 +41,20 @@ release: voboot
 apidoc: voboot
 	dune build $(DUNEOPT) @doc
 
+DUNE_BOOT_VER=1.3.0
+DUNE_TBZ=dune-$(DUNE_BOOT_VER).tbz
+DUNE_BOOT_BIN=_dune/dune-$(DUNE_BOOT_VER)/_build/install/default/bin/dune
+bootstrap:
+	mkdir -p _dune
+	( cd _dune && wget https://github.com/ocaml/dune/releases/download/$(DUNE_BOOT_VER)/$(DUNE_TBZ) && tar xvfj $(DUNE_TBZ) )
+	( cd _dune/dune-$(DUNE_BOOT_VER)/ && make release )
+	@echo "A Local version of Dune is ready to be used, add _dune/dune-$(DUNE_TBZ)/_build/install/default/bin/ to your PATH"
```

Comment:
Typo: `DUNE_TBZ` should be `DUNE_BOOT_VER`.

Reply:
Indeed, thanks.